web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -36,13 +41,24 @@ function StorePlugin(props: StorePluginProps) {
     }
   }, [isInstallSuccess]);
 
+  const topActions = [
+    <div className="relative whitespace-nowrap">
+      <Tooltip
+        content={warningMessage}
+        icon={<FiAlertTriangle className="text-s-warning" size={24} />}
+      />
+    </div>,
+    <FiDownload className="w-6 h-10 text-tertiary" />,
+  ];
+
   const argsStore = {
     preIcon: <img src={logo} alt="plugin-logo" />,
     topAction: <FiDownload onClick={() => mutate({ params })} />,
     title: name,
     subtitle: author,
     subtitleIcon: massalabsNomination.includes(author) ? <Certificate /> : null,
     content: description,
+    topActions: !iscompatible ? topActions : undefined,
```

Comment:
```suggestion
    topActions: isIncompatible ? topActions : undefined,
```

Reply:
I'm not sure about this, it changes the initial logic

Reply:
All good we synced in a call!